data/brands/amenity/bicycle_rental.json

```diff
@@ -1524,29 +1524,72 @@
       }
     },
     {
-      "displayName": "Santander Cycles",
+      "displayName": "Santander Cycles (Brunel)",
+      "locationSet": {"include": ["gb-eng"]},
+      "tags": {
+        "amenity": "bicycle_rental",
+        "brand": "Santander Cycles",
+        "brand:wikidata": "Q807961",
+        "brand:wikipedia": "en:Santander Cycles",
+        "network": "nextbike",
+        "operator": "nextbike",
+        "operator:wikidata": "Q2351279",
+        "operator:wikipedia": "en:nextbike"
+      }
+    },
+    {
+      "displayName": "Santander Cycles (London)",
       "id": "serco-b88dc4",
       "locationSet": {"include": ["gb-eng"]},
       "matchNames": [
         "barclays bikes",
         "barclays cycle hire",
         "boris bike",
-        "boris bikes",
-        "santander cycles mk"
+        "boris bikes"
       ],
       "note": "Formerly sponsered by 'Barclays Bank', hence the matchNames.",
       "tags": {
         "amenity": "bicycle_rental",
         "brand": "Santander Cycles",
         "brand:wikidata": "Q807961",
         "brand:wikipedia": "en:Santander Cycles",
-        "name": "Serco",
         "network": "tfl_cycle_hire",
         "operator": "Serco",
         "operator:wikidata": "Q499104",
         "operator:wikipedia": "en:Serco"
       }
     },
+    {
+      "displayName": "Santander Cycles (Milton Keynes)",
+      "locationSet": {"include": ["gb-eng"]},
+      "matchNames": [
+        "santander cycles mk"
+      ],
+      "tags": {
+        "amenity": "bicycle_rental",
+        "brand": "Santander Cycles MK",
+        "brand:wikidata": "Q807961",
+        "brand:wikipedia": "en:Santander Cycles",
+        "network": "nextbike",
+        "operator": "nextbike",
+        "operator:wikidata": "Q2351279",
+        "operator:wikipedia": "en:nextbike"
```

Comment:
Their tagging is a bit random:
https://overpass-turbo.eu/s/10Rf

Some use name or network.

Including docking station number in the name rather than in the ref.